Gemfile

```diff
@@ -22,7 +22,9 @@ group :docs do
   gem 'kramdown' # Markdown implementation (for yard)
 end
 
-gem "rails", "~> 5.2.2"
-gem "activerecord-jdbcsqlite3-adapter", "~> 52.0", platform: :jruby
+# Use the unreleased version of inherited_resources because is compatible with Rails 6
+gem 'inherited_resources', github: 'activeadmin/inherited_resources'
+gem "rails", "6.0.0.beta2"
+gem "activerecord-jdbcsqlite3-adapter", github: 'jruby/activerecord-jdbc-adapter', platform: :jruby
```

Comment:
I think we can follow the same strategy as in https://github.com/activeadmin/inherited_resources/pull/479. Let's leave the main Gemfile intact for now, and tackle Rails 6.0 on a different gemfile, for example, `gemfiles/rails_60.gemfile{,.lock}`.

Reply:
I separated the Gemfile for Rails 6 @deivid-rodriguez